CHANGELOG.md

```diff
@@ -1,4 +1,100 @@
-<a name="9.10.0></a>
+<a name="10.0.0"></a>
+
+# [10.0.0](https://github.com/commercetools/ui-kit/compare/v9.10.0...10.0.0) (2019-06-20)
+
+### Breaking Changes
+
+This release introduces **breaking changes** which may entail some **migration steps**. We'll go through them now:
+
+- `hocs`: `withMouseOverState` and `withMouseDownState` have been dropped. Please use CSS properties instead. ([#881](https://github.com/commercetools/ui-kit/pull/881))
+
+- `Icons`: make icons themeable (breaking change) ([#726](https://github.com/commercetools/ui-kit/issues/726)) ([9bea3ad](https://github.com/commercetools/ui-kit/commit/9bea3ad))
+
+The prop `theme` has been renamed to `color` for Icons. Also, the values of this prop have changed. Please see the table below for the mapping between the old and new values.
```

Comment:
It would be good to provide a codemod for this kind of changes.

Reply:
We already added a codemod in `10.0.0` which can be used for this: https://github.com/commercetools/ui-kit/blob/10.0.0/transforms/react-component-rename-props.js